src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -607,7 +607,9 @@ public function formAction($id, Request $request)
             // so we need to return the right type of response if an exception it thrown
             if ($request->isXmlHttpRequest()) {
                 return $this->returnErrorJsonResponse(
-                    [],
+                    // this is not perfect, but if there was an exception during saving or in a hook
+                    // atleast show the message near the name to help user identify what happened
+                    ['step1_name_1' => [$e->getMessage()]],
```

Comment:
Question : Why 'step1_name_1' as key name ?

Reply:
It is shown under the product name input field.

There is no "generic" place for these errors available in the template (at least I didn't find one easily).